docs/src/roadmap.md

```diff
@@ -40,7 +40,7 @@ Some but not all of these tasks are summarized in the
   - Get JuMP’s unit tests running in less than two minutes. See [#1745](https://github.com/jump-dev/JuMP.jl/pull/1745).
 - All solvers should complete the transition to MOI.
 - Provide packages for installing Bonmin and Couenne.
```

Comment:
Is https://github.com/GAMS-dev/gams.jl a solution for this?

Reply:
Not really since I believe it requires a GAMS license.